app/src/main/java/com/td/mylevelup/mortgage/mortgageCalculatorPage/MortgageActivity.kt

```diff
@@ -16,13 +19,14 @@ import com.td.mylevelup.Constants.Companion.MAX_MORTGAGE_AMOUNT
 import com.td.mylevelup.Constants.Companion.MIN_AMORTIZATION_PERIOD
 import com.td.mylevelup.Constants.Companion.MIN_MORTGAGE_AMOUNT
 import com.td.mylevelup.R
+import com.td.mylevelup.mortgage.mortgageResultsPage.MortgagePaymentResultActivity
 import com.thejuki.kformmaster.helper.FormBuildHelper
 import com.thejuki.kformmaster.model.BaseFormElement
 import com.thejuki.kformmaster.model.FormLabelElement
 import com.thejuki.kformmaster.model.FormNumberEditTextElement
 import java.math.BigDecimal
 
-class MortgageActivity: AbstractActivity<MortgagePagePresenter, MortgagePageAdapter>(), MortgagePageView {
+class MortgageActivity : AbstractActivity<MortgagePagePresenter, MortgagePageAdapter>(), MortgagePageView {
```

Comment:
extra space after `MortgageActivity`, the colon should have no space. It's `activity: AbstractActivity`